index.test.js

```diff
@@ -210,3 +211,17 @@ test('reports 0 coverage when no lines are found ', () => {
     expect(errorMessage).toContain('0 is less than min_coverage 100');
   }
 });
+
+test('fails when min_coverage is not a number', () => {
+  process.env['INPUT_MIN_COVERAGE'] = '10%';
```

Comment:
If we go ahead with https://github.com/VeryGoodOpenSource/very_good_coverage/pull/290/files#r1360360519, then we should update this to something that can't be parsed by [parseInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt).

```suggestion
  process.env['INPUT_MIN_COVERAGE'] = 'xyz';
```